src/Command/CommandGenerator.php

```diff
@@ -47,9 +47,10 @@ public function sshCommand(string $destination, string $currentHost, string $com
      * @param string $from
      * @param string $to
      * @param string $currentHost
+     * @param bool $all
      * @return string[]
      */
-    public function syncCommands(string $from, string $to, string $currentHost): array
+    public function syncCommands(string $from, string $to, string $currentHost, bool $all): array
```

Comment:
i think the bool $all should have a default value, per default it should sync everything